doc/src/devdocs/require.md

```diff
@@ -0,0 +1,44 @@
+# Base.require
+
+`Base.require` is responsible for loading modules it also manages the
```

Comment:
some sort of punctuation between "is responsible for loading modules" and "it also manages"